bin/generate_repos.py

```diff
@@ -55,14 +55,16 @@ def add_empty_commit(folder,which_branch):
     # otherwise clone to a non-bare repo and add an empty commit
     # to the specified branch
     with tempfile.TemporaryDirectory() as tmpdirname:
-        os.system(f'git clone {folder} {tmpdirname}')
+        subprocess.run(['git', 'clone', folder, tmpdirname])
         os.chdir(tmpdirname)
-        os.system(f'git checkout -b {which_branch}')
-        os.system("git " +
-                  "-c user.name=submitty -c user.email=submitty@example.com commit " +
-                  "--allow-empty -m 'initial empty commit' " +
-                  "--author='submitty <submitty@example.com>'")
-        os.system(f'git push origin {which_branch}')
+        subprocess.run(['git', 'checkout', '-b', which_branch])
+        subprocess.run(['git',
+            '-c', 'user.name=submitty',
+            '-c', 'user.email=submitty@example.com',
+            'commit', '--allow-empty',
+            '-m initial empty commit',
```

Comment:
Should this not be two different arguments?

```suggestion
            '-m', 'initial empty commit',
```

Reply:
Both ways work. I had changed it to the current way to resolve a bug I thought it was causing but it seems I can change it back.